adminpages/reports/login.php

```diff
@@ -252,194 +273,265 @@ function pmpro_report_login_page()
 /*
 	Other code required for your reports. This file is loaded every time WP loads with PMPro enabled.
 */
-
-//track visits
-function pmpro_report_login_wp_visits()
+//get values for a user
+function pmpro_login_report_get_values_for_user($type, $user_id = NULL)
 {
-	//don't track admin

```

Comment:
we prob want this back if it's not still in there.

Reply:
This is the function that would replace the pmpro_report_login_wp_visits - https://github.com/strangerstudios/pmpro-better-logins-report/blob/master/reports/better-logins.php#L390

Would this be okay to replace or should we keep this function for tracking visits?

Reply:
This function pmpro_report_login_wp_visits will become a wrapper like in BLR:

```
/track visits
function pmpro_report_better_login_wp_visits()
{
	pmpro_report_login_track_values("visits");	
}
add_action("wp", "pmpro_report_better_login_wp_visits");
```

The pmproblr_trackValues renamed to pmpro_report_login_track_values and use code from BLR